homeassistant/components/zabbix/sensor.py

```diff
@@ -23,11 +24,17 @@
         vol.Optional(CONF_NAME): cv.string,
     }
 )
+_ZABBIX_ITEM_SCHEMA = vol.Schema(
+    [{vol.Optional(CONF_NAME): cv.string, vol.Required(CONF_ID): cv.string}]
+)
 
 # SCAN_INTERVAL = 30
 #
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(_CONF_TRIGGERS): vol.Any(_ZABBIX_TRIGGER_SCHEMA, None)}
+    {
+        vol.Optional(_CONF_TRIGGERS): vol.Any(_ZABBIX_TRIGGER_SCHEMA, None),
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler